bzt/jmx2yaml.py

```diff
@@ -1122,7 +1119,7 @@ def process_tg(self, tg_etree_element):
                     try:
                         request['body'] = json.loads(body)
                     except (ValueError, TypeError):
-                        raise TaurusInternalException("Following body cannot be converted into JSON:\n%s", body)
+                        self.log.warning("Following body cannot be converted into JSON:\n%s", body)
```

Comment:
Is this body skipped or added as is? In my point we need to clarify that. Is it possible to add useful error message? (e.g. 'Error in 0xXX position')